src/sb_pointcloud_processing/test/test_scripts/nodelet_launch_rostest.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
```

Comment:
We should add a descriptive comment here explaining what this file is doing and why it exists.

Reply:
Basically the same purpose as `nodelet_manager.sh/nm.sh`. Marked it in this document as well :heavy_check_mark: